api/src/utils/get-accountability-for-token.ts

```diff
@@ -23,8 +23,8 @@ export async function getAccountabilityForToken(
 			const payload = verifyAccessJWT(token, env['SECRET'] as string);
 
 			accountability.role = payload.role;
-			accountability.admin = payload.admin_access === true || payload.admin_access == 1;
-			accountability.app = payload.app_access === true || payload.app_access == 1;
+			accountability.admin = payload.admin_access === true || payload.admin_access === 1;
+			accountability.app = payload.app_access === true || payload.app_access === 1;
```

Comment:
May also need confirmation here. I sort of recall noticing this in the past, but it's technically correct as `admin_access` and `app_access` might be a string for SQLite, but I could be wrong here.

But understandably this isn't very obvious at a glance, so if we do end up remaining them as loose equality, let's also add comment for this.